CodeScanList.json

```diff
@@ -3131,6 +3131,17 @@
                 "All": true
             }
         },
+        "Mirror.Transports": {
+            "Mirror.MultiplexTransport": {
+                "All": true

```

Comment:
due to the security risk only poke holes what you need

Reply:
This is exactly what Codescan is complaining about
you posted the errors

Reply:
well as in, you don't need to access everything, you only need to add access to the stuff you're using, yyacht using every single variable inside of Mirror.MultiplexTransport

Reply:
can remove the         "All": true from them and it should give a bit more detailed error